packages/coinstac-docker-manager/package.json

```diff
@@ -9,13 +9,16 @@
   "author": "",
   "license": "ISC",
   "dependencies": {
+    "clarify": "^2.1.0",
     "coinstac-socket.io-stream": "^0.9.1",
     "dockerode": "^2.5.5",
     "lodash": "^4.17.10",
     "portscanner": "^2.2.0",
     "request-stream": "^1.2.2",
-    "socket.io-client": "^2.1.1",
+    "socket.io-client": "^2.3.0",
```

Comment:
is this being used?

Reply:
nah, there is a couple more things that need to be pruned too